src/common-ui/Text.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { render, screen } from '@testing-library/react';
+import Text from './Text';
+
+describe('Text test', () => {
+  test('Text 컴포넌트는 children을 렌더링한다.', () => {
+    //given
+    const TEXT = 'Hello World';
+
+    //when
+    render(
+      <Text type={{ tag: 'h1', type: 'header' }} weight="bold">
+        {TEXT}
+      </Text>,
+    );
+
+    //then
+    expect(screen.getByText(TEXT)).toBeInTheDocument();
+  });
+  test('Text 컴포넌트는 type에 따라 태그를 렌더링한다.', () => {
+    //given
+    const TEXT = 'Hello World';
+
+    //when
+    render(
+      <Text type={{ tag: 'h2', type: 'header' }} weight="bold">
+        {TEXT}
+      </Text>,
+    );
+
+    //then
+    expect(screen.getByRole('heading', { level: 2 })).toBeInTheDocument();
```

Comment:
type에 따른 렌더링을 테스트하는 것이니,  given 절에 테스트 대상 조건인 TYPE 을 선언하는것이 어떨까용??

```suggestion
    //given
    const TYPE = {tag:'h2', type:'header'}
    
    //when
    render(
      <Text type={TYPE} weight="bold">
          Text
      </Text>,
    )

    //then
    expect(screen.getByRole('heading', { level: 2 })).toBeInTheDocument();
```

Reply:
오 이렇게 수정해 보겠습니다😃